### ISSUE TITLE
Change 'Done' to 'Submit'

### ISSUE THREAD

**Issue by beckyrother** [zooniverse/Panoptes-Front-End#5681]
## Expected behavior
The current wording is confusing and does not create a lot of confidence in the submission of classifications. Changing to Submit makes it much more clear that a volunteer is Submitting a classification. 

I'd like to get the ball rolling on this change. @mrniaboc what type of communication to project owners will this require, and can we allow projects to opt in or out? For some teams this will require creating new screenshots for their tutorials (but that's another question).

**Comment by mrniaboc** [comment]
We would definitely need to notify project owners letting them know our
plans and when it would be happening, asking them to make the appropriate
changes to their tutorials, help text etc. We will also need to edit our
own materials, like the project building docs. Finally, we'll need to
update the translations for languages who have translated the general
Zooniverse platform text.

**Comment by eatyourgreens** [comment]
Translations will be need to be updated in the language files here: https://github.com/zooniverse/Panoptes-Front-End/tree/master/app/locales

**Comment by srallen** [comment]
Update will need to be done in the new classifier as well: https://github.com/zooniverse/front-end-monorepo/tree/master/packages/lib-classifier/src/components/Classifier/components/TaskArea/components/Tasks/components/TaskNavButtons/components

**Comment by chrislintott** [comment]
Could we do it in the new classifier first, measure the change in behaviour on PH and then use that to reassure nervous project owners on the rest of the sites?

C

On 14 Apr 2020, at 20:46, srallen <notifications@github.com> wrote:

﻿

Update will need to be done in the new classifier as well: https://github.com/zooniverse/front-end-monorepo/tree/master/packages/lib-classifier/src/components/Classifier/components/TaskArea/components/Tasks/components/TaskNavButtons/components

**Comment by beckyrother** [comment]
Yes, that would be a good test as long as the PH team are ok with the change?

**Comment by seanmiller26** [comment]
I’d be in favor of an A/B test on a specific FEM project so certain users at random see one button or the other. We would have to communicate this is happening and have a link to a survey to get volunteers' thoughts.
